### ISSUE TITLE
I can't see the cats in the first tutorial

### ISSUE THREAD

**Issue by rod-farias** [kittykatattack/hexi#76]
I can't see the cats in the first tutorial on Firefox 90.0 (64-bit), Linux Mint 20.1. However, Treasure Hunter and Alien Armada display correctly.

![Hexi quick start - Mozilla Firefox_004](https://user-images.githubusercontent.com/57115850/127327824-743bb924-9754-4a39-b691-36579c5c2ec4.png)

**Comment by malceore** [comment]
What's the output in your Javascript Console?
https://developer.mozilla.org/en-US/docs/Tools/Browser_Console

Edit: didn't see you were hitting a CDN and not local.

**Comment by rod-farias** [comment]
https://cdn.gitcdn.xyz/cdn/kittykatattack/hexi/a1713aa19bdcc9a0c661e67d079a205d7c221917/examples/01_quickStart.html

![Selección_001](https://user-images.githubusercontent.com/57115850/127349385-e101d2c5-1547-4d9c-915b-612a49a45ed7.png)

![Selección_002](https://user-images.githubusercontent.com/57115850/127349432-0d924039-580c-4ebd-9214-5701c87addb6.png)

![Selección_003](https://user-images.githubusercontent.com/57115850/127349453-606e9e69-5be0-4e64-b86f-6b84fab2d091.png)

**Comment by malceore** [comment]
Yeah I can recreate the behavior, seems like a bug. Maybe I'll see about fixing it and opening up a PR.
